.github/actions/setup-binary-builds/action.yml

```diff
@@ -33,7 +33,7 @@ runs:
   using: composite
   steps:
       - name: Remove repository directory (if exists)
-        shell: bash
+        shell: bash -l {0}
```

Comment:
What does this do?

Reply:
It invokes the shell as a login shell that allows us to pick up changes that may be made to bash profile files by `conda init bash`: https://github.com/conda-incubator/setup-miniconda#important. Without this, conda may not be found in subsequent steps even after installation.